Dockerfile

```diff
@@ -493,6 +499,9 @@ ENV GEM_HOME=/var/tmp/gem
 ENV PATH=$PATH:$RBENV_ROOT/bin:$RBENV_ROOT/shims:$RBENV_ROOT/plugins/ruby-install/bin
 COPY --from=ruby --chown=$USER:$USER $RBENV_ROOT $RBENV_ROOT
 
+# ScanCode license texts directory
```

Comment:
Still missing a dot at the end of this sentence 😉

Reply:
This is not a sentence, because it has no verb :)

Reply:
Removed the comment completely, because it is rather obvious what is happening there.